src/libraries/System.Private.CoreLib/src/System.Private.CoreLib.Shared.projitems

```diff
@@ -193,6 +193,7 @@
     <Compile Include="$(MSBuildThisFileDirectory)System\Decimal.DecCalc.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)System\DefaultBinder.cs" />
     <Compile Include="$(MSBuildThisFileDirectory)System\Delegate.cs" />
+    <Compile Include="$(MSBuildThisFileDirectory)Internal\Diagnostics\CodeAnalysis\DynamicallyAccessedMembersOnThisAttribute.cs" />
```

Comment:
Nit: We are keeping this list alphabetically sorted. It should be in the `Internal*` block.